dbos/_dbos_config.py

```diff
@@ -310,6 +311,17 @@ def load_config(
 
     """
 
+    # Get the current call stack
```

Comment:
Look who's back!  Still has the same bug too...

Reply:
yeah a merge w/ main brought that in. will fix